content/blog/2022-01-14-users-guide-to-contribution-ross/index.md

```diff
@@ -1,12 +1,13 @@
 ---
 title: "A contributor's guide to the Galaxy"
-tease: "Contribution opportunities"
+tease: "Contribution opportunities for community members"
```

Comment:
Do you think that this post is useful only if you're already a member? I'd say that some are good mechanisms to _become_ a member

Reply:
Yes, maybe best to revert to what it was - will do now